app/views/customers/index.html.erb

```diff
@@ -1,18 +1,18 @@
-<h1>Listagem de clientes</h1>
+<h1><%= t('.title') %></h1>
 
 <table>
   <thread>
     <tr>
-      <th>Nome</th>
-      <th>CPF</th>
+      <th><%= t('.list.name') %></th>
+      <th><%= t('.list.cpf') %></th>
```

Comment:
Well… I don't think so… you are using an attribute from vehicle…

Answer this question:

What happen if you have two tables… one for vehicle and another for order, in both you will display the color of the vehicle, but in vehicle table you name it _color_ and in order table you name it _vehicle's color_…

?